uitools/common/src/LocatorSearchSource.h

```diff
@@ -16,19 +16,19 @@
 #ifndef ESRI_ARCGISRUNTIME_TOOLKIT_LOCATORSEARCHSOURCE_H
 #define ESRI_ARCGISRUNTIME_TOOLKIT_LOCATORSEARCHSOURCE_H
 
-// Toolkit headers
-#include "SearchSourceInterface.h"
+// Qt headers
+#include <QFuture>
+#include <QList>
+#include <QObject>
 
-// ArcGISRuntime headers
+// STL headers
```

Comment:
These are wrong, too. Just comments though, so we can do another sweep or you can find/fix all the offenders in this PR (easier said than done).